src/libertem/web/base.py

```diff
@@ -104,3 +106,21 @@ async def send_existing_job_results(self):
             await self.send_results(
                 result_set, job_id, analysis_id, details, finished=True
             )
+
+def handle_session(reqHandler, state, event_registry):
+    if  reqHandler.get_secure_cookie("SESSID") is not None and reqHandler.get_secure_cookie("SESSID").decode("utf-8") in state.get_sessions():
+        reqHandler.state = state.get_sessions()[reqHandler.get_secure_cookie("SESSID").decode("utf-8")]["SessionState"]
+    else:
+        # Generate a random session id
+        id = secrets.token_hex(32)
+        reqHandler.state = state.register_session(id)
+        reqHandler.set_secure_cookie("SESSID", id, expires_days=None)
+    reqHandler.event_registry = reqHandler.state.event_registry
+
+class SessionsWebSocket:
```

Comment:
I override the initialization function for all the handlers throughout the server.